### PR TITLE
Quick Disable Third-Party Libraries

### PR DESCRIPTION
**Pull request by icirellik**
Description

This change adds an environment variable that makes it easy to quickly
disable all third party libraries without needing to know all of the
variables to unset. This greatly simplifies local performance tests by
removing the variability that is created by these third-party tests.
This flag should only be used for local testing.

View without whitespace for a much simpler diff:
https://github.com/artsy/force/pull/6268/files?w=1

### CODE REVIEW

**Comment by dzucconi** [comment]
THANK YOU

File: src/typings/sharify.d.ts [hunk 1]
```diff
@@ -89,6 +90,7 @@ declare module "sharify" {
       SUBMISSION_ARTIST_NAME: string
       TARGET_CAMPAIGN_URL: string
       TRACK_PAGELOAD_PATHS: string
+      THIRD_PARTIES_DISABLED: string
```
Comment (joeyAghion) [inline src/typings/sharify.d.ts:93]: Why not `boolean`?

File: src/desktop/components/main_layout/templates/scripts.jade [hunk 2]
```diff
@@ -159,11 +162,11 @@ if options.sailthru && sd.SAILTHRU_CUSTOMER_ID
     })();
 
 //- Prefetch links on hover. See: https://instant.page/
-if sd.ENABLE_INSTANT_PAGE
+if sd.ENABLE_INSTANT_PAGE && !sd.THIRD_PARTIES_DISABLED
   include instant-page
```
Comment (joeyAghion) [inline src/desktop/components/main_layout/templates/scripts.jade:166]: This one is a little surprising. Isn't this a client-side script we choose to include for its benefits to the perceived loading times? If it's not helpful, we could choose to eliminate it, as opposed to the truly "third party" scripts that are needed for non-technical tracking/integration reasons.

**Review by joeyAghion — APPROVED** [review]

File: src/typings/sharify.d.ts (hunk 1 above)
Comment (icirellik) [reply src/typings/sharify.d.ts:93]: No reason, the few "boolean-ish" values I looked at were strings, I see there are some booleans so I will update it.

File: src/desktop/components/main_layout/templates/scripts.jade (hunk 2 above)
Comment (icirellik) [reply src/desktop/components/main_layout/templates/scripts.jade:166]: Let me do some research as I'm not entirely sure what this script is for or even if we have it enabled.

**Pull request merged**